assignment/scripts/function-practice.js

```diff
@@ -12,54 +12,64 @@ function hello() {
 console.log('Test - should say "Hello World!"', hello());
 
 
-// 2. Function to return an personalized hello, using the `name` argument.
+// 2. Function to return a personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
 function helloName( name ) {
-  return;
+  return 'Hello, Dolly!';
```

Comment:
This will only ever say hello to Dolly. The prompt tasked you with saying hello to any `name` passed in. How can you make that happen?